docs/source/tutorials/text.ipynb

```diff
@@ -11,15 +11,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this 5-minute quickstart tutorial, we use cleanlab to find potential label errors in a text classification dataset of [IMDB movie reviews](https://ai.stanford.edu/~amaas/data/sentiment/). This dataset contains 50,000 text reviews, each labeled with a binary sentiment polarity label indicating whether the review is positive (1) or negative (0). cleanlab will shortlist _hundreds_ of examples that confuse our ML model the most; many of which are potential label errors, edge cases, or otherwise ambiguous examples.\n",
+    "In this 5-minute quickstart tutorial, we use cleanlab to find potential label errors in an intent classification dataset composed of (text) customer service requests at an online bank. We consider a subset of the [Banking77-OOS Dataset](https://arxiv.org/pdf/2003.04807.pdf) containing 1,000 customer service requests which can be classified into 10 categories corresponding to the intent of the request. cleanlab will shortlist examples that confuse our ML model the most; many of which are potential label errors, out-of-scope examples, or otherwise ambiguous examples.\n",
```

Comment:
I believe the dataset is just called Banking77? Would just use the official name, so stick with current one if that's the official one and I am wrong here.

Reply:
Banking77-OOS is a variation of the original Banking77 dataset with added out-of-scope examples. I believe that is the dataset @ulya-tkch curated from. This link points to the wrong paper (current one introduces Banking77 instead of Banking77-OOS), will swap out this link, but otherwise dataset name is correct.